liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java

```diff
@@ -664,6 +664,8 @@ private void configureLogging(Level logLevel, String logFile) throws IOException
 
         java.util.logging.Logger rootLogger = java.util.logging.Logger.getLogger("");
         Level cliLogLevel = logLevel;
+        // If a log file is set, but no log level is set, this variable represents the log level that should be used.
```

Comment:
Do we also need to check `logFormat` here?

Reply:
Nope, this applies to all log formats. This was existing functionality that has gradually broken over the years.